doc/tutorials/scripts/create_up2_images.sh

```diff
@@ -0,0 +1,283 @@
+#!/bin/bash
+
+
+usage() {
+    echo "Usage: $0 [options]"
+    echo "This script build images for SBL based platforms"
+    echo "options:"
+    echo "--mirror-url default: 'https://cdn.download.clearlinux.org/releases/', for swupd"
+    echo "--acrn-code-path: Specify acrn_hypervisor code path for acrn sbl build. If acrn-sbl-path is provided, acrn-code-path will be ignored"
+    echo "--acrn-sbl-path: Specify acrn sbl binary path. If acrn-sbl-path isn't provided, acrn_code_path will be must option"
+    echo "--clearlinux-version: mandatory option for sos images build"
+    echo "--images-type: Specify the type of OS image to build (sos/laag/all, default vaule is all)"
+    echo "--sign-key: Specify the debug key for signing, default value provided"
+    echo "--sos-rootfs-size: Specify the sos_rootfs image size, default value is 3584M"
+    echo "--laag-image-size: Specify the laag image size, default value is 20480M"
+    echo "--sos-bundle-append: Specify more sos bundles need to be add"
+    echo "--laag-json: mandatory option for uos image build, used for ister.py"
+}
+
+    
+create_sos_images() {
+    mkdir sos_rootfs
+    echo "Clean previously generated images"
+    rm -fv sos_boot.img
+    rm -fv sos_rootfs.img
+
+    fallocate -l ${SOS_ROOTFS_SIZE}M sos_rootfs.img || return 1
+    mkfs.ext4 sos_rootfs.img
+    mount sos_rootfs.img sos_rootfs
+    echo mount sos_rootfs >> .cleanup
+
+    mountpoint sos_rootfs || return 1
+
+    swupd verify --install --path=sos_rootfs --contenturl=$MIRRORURL --versionurl=$MIRRORURL --format=staging -m ${VERSION} ||
+    {
+        echo "Failed to swupd install"
+        return 1
+    }
+
+    swupd bundle-add $SOS_BUNDLE_LIST --path=sos_rootfs --contenturl=$MIRRORURL --versionurl=$MIRRORURL --format=staging ||
+    {
+        echo "Failed to swupd bundle add"
+        return 1
+    }
+
+    SOS_BOOTARGS_DEBUG=${ACRN_HV_CODE_PATH}/devicemodel/samples/up2/sos_bootargs_debug.txt
+
+    if [[ ! ${ACRN_SBL} && ! -f ${ACRN_SBL} ]]
+    then 
+        if [ ${ACRN_HV_CODE_PATH} ]
+        then
+            make -C ${ACRN_HV_CODE_PATH} clean || return 1
+            make -C ${ACRN_HV_CODE_PATH} hypervisor PLATFORM=sbl BOARD=up2 FIRMWARE=sbl || return 1

```

Comment:
Should we already remove the `PLATFORM=sbl` parameter here since it's being deprecated in the codebase?